src/ray/stats/metric_defs.cc

```diff
@@ -156,16 +156,19 @@ DEFINE_stats(io_context_event_loop_lag_ms,
              ray::stats::GAUGE);
 
 /// Event stats
-DEFINE_stats(operation_count, "operation count", ("Method"), (), ray::stats::GAUGE);
-DEFINE_stats(
-    operation_run_time_ms, "operation execution time", ("Method"), (), ray::stats::GAUGE);
-DEFINE_stats(
-    operation_queue_time_ms, "operation queuing time", ("Method"), (), ray::stats::GAUGE);
-DEFINE_stats(operation_active_count,
-             "activate operation number",
-             ("Method"),
+DEFINE_stats(operation_count, "operation count", ("Name"), (), ray::stats::COUNT);
+DEFINE_stats(operation_run_time_ms,
+             "operation execution time",
+             ("Name"),
              (),
-             ray::stats::GAUGE);
+             ray::stats::HISTOGRAM);
```

Comment:
where do we define the buckets for the histogram?

(i'm not that familiar with C++ metrics API)

Reply:
from `grpc_server_req_process_time_ms` below, I think it needs to be in the fourth arg to the `DEFINE_stats` macro

Reply:
What's a good granularity for this?  I think I recall seeing a standardized granularity somewhere in the codebase, but I'm struggling to find where I found that just now...

Reply:
{1, 10, 100, 1000, 10000} looks pretty standard?  I'll put that up for now.

Reply:
what's the unit? if those are in ms that looks good to me